### DISCUSSION TITLE
Add Lyrics Tab

### DISCUSSION THREAD

**Discussion by adilurrhmn** [MissingCore/Music#75] [Ideas]
I can't find Embedded lyrics option.
Please add it in next update

**Comment by cyanChill** [comment]
I wrote a document; [App Goals & Ideology](https://github.com/MissingCore/Music/blob/main/docs/app-goals-ideology.md) which goes over a bit on what I want this app to be along with things I don't want to add.

Lyrics are one of the things that I don't want to add due cost & legal stuff (lyrics are considered Intellectual Property). Since this app is also going to be on the Play Store, its more of a risky thing since Google has my info and I don't want to get into any legal trouble. If I wasn't going to publish this on the Play Store, I would consider finding a way to do this.

**Comment by adilurrhmn** [comment]
I am taking about offline lyrics. Maximum music player support offline lyrics abd also available in Play Store.
